### PR TITLE
merged changes from master and add tests for secondary sitetype

### PR DESCRIPTION
**Pull request by abhijitb**

### CODE REVIEW

**Review by arunshenoy99 — APPROVED** [review]
We can loop through all the categories using `each` in future iterations.

File: tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/sitetype-secondary.cy.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+// <reference types="Cypress" />
+
+describe('Get Started Site Type Secondary', function () {
+
+    before(() => {
+        cy.setCustomerData();
+        cy.visit('wp-admin/?page=nfd-onboarding&flow=ecommerce#/wp-setup/step/get-started/site-secondary');
+        // cy.injectAxe(); 
+        cy.wait(5000);
```
Comment (arunshenoy99) [inline tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/sitetype-secondary.cy.js:9]: Can we reduce this wait time to maybe `2000-3000`?

**Review by arunshenoy99 — CHANGES_REQUESTED** [review]
One comment to reduce the time, rest all seem to be fine to me, if you have time we can modify the code to loop through all the boxes as well (Ref products.cy.js) but that is optional.

**Review by arunshenoy99 — APPROVED** [review]
Looks good thanks AB.

**Review by officiallygod — APPROVED** [review]

**Pull request merged**